widgets/google-map/google-map.php

```diff
@@ -122,6 +121,21 @@ function get_widget_form(){
 						'integer'     => true,
 
 					),
+
+					// Mobile Zoom is hidden for static maps as there's no reliable way to check for mobile devices that will bypass caching
```

Comment:
Couldn't we check for this in the JS and adjust the static image's URL accordingly?